loops.js

```diff
@@ -16,9 +19,11 @@
 // loop...
 // loop...
 // klaar!
- // ==========================================
-
-
+// ==========================================
+for (let i = 0 ; i < 4; i++ ) {
```

Comment:
De code geeft de verwachte uitkomst, echter in de opdracht staat dat de loop 5 keer moet lopen ipv 4 keer.
Hier wordt de code 4x ge-looped en wanneer de loop klaar is wordt "klaar!"gelogd. Dit zou in de loop moeten gebeuren bijvoorbeeld: for (let i = 0; i < 5; i++) {
if (i < 4) {
console.log("loop...")
} else {
console.log("klaar");
}
}